controllers/rest/user.go

```diff
@@ -151,3 +153,27 @@ func (c *Controller) SetPaypalMailAddr(session session.Manager) http.HandlerFunc
 		return
 	}
 }
+
+func (c *Controller) GetUser() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		id, err := strconv.Atoi(chi.URLParam(r, "id"))
+
+		if err != nil {
+			respond(w, r, http.StatusUnprocessableEntity, ErrInvalidNumberFormat.Error())
+			return
+		}
+
+		publicUser, err := c.userService.Get(uint64(id))
+
+		if err != nil && err == offer.ErrOfferNotFound {
```

Comment:
Wrong error, should be `user.ErrUserNotFound` :D

Reply:
you are right